dataikuapi/dss/webapp.py

```diff
@@ -0,0 +1,128 @@
+import sys
+from .future import DSSFuture
+
+if sys.version_info >= (3,0):
+  import urllib.parse
+  dku_quote_fn = urllib.parse.quote
+else:
+  import urllib
+  dku_quote_fn = urllib.quote
+
+
+class DSSWebApp(object):
+    """
+    A handle to manage a webapp
+    """
+    def __init__(self, client, project_key, webapp_id):
+        """Do not call directly, use :meth:`dataikuapi.dss.project.DSSProject.get_webapps`"""
+        self.client = client
+        self.project_key = project_key
+        self.webapp_id = webapp_id
+        self.state = None
+
+    def get_state(self):
+        """
+        Return the state of the webapp
+
+        :return: the state of the webapp
+        :rtype: :class:`DSSWebAppBackendState`
+        """
+        if self.state is None:
```

Comment:
I would not store/cache the state of the backend, otherwise, the user cannot get a refreshed state for one instance of a DSSWebApp (calling get_state several times will always result of the same state)